libraries/ts-command-line/src/providers/CommandLineParameterProvider.ts

```diff
@@ -87,6 +87,12 @@ const LONG_NAME_GROUP_NAME: string = 'longName';
 const POSSIBLY_SCOPED_LONG_NAME_REGEXP: RegExp =
   /^--((?<scope>[a-z0-9]+(-[a-z0-9]+)*):)?(?<longName>[a-z0-9]+((-[a-z0-9]+)+)?)$/;
 
+interface IExtendedArgumentGroup extends argparse.ArgumentGroup {
+  // The types are incorrect - this function returns the constructed argument
```

Comment:
What does this return according to the type?

Reply:
The types say it returns `void`

Reply:
Oh that's silly